functions/backup.bash

```diff
@@ -446,7 +446,7 @@ setup_mirror_SD() {
   local dest
   local srcSize
   local destSize
-  local minBackupSize
+  local minStorageSize=4294967296                   # 4 GB
```

Comment:
I don't mean to give you a headache, but because of the nasty people in charge of marketing there is a difference between GiB and GB:

![Screen Shot 2021-02-26 at 10 18 45 AM](https://user-images.githubusercontent.com/15617561/109332935-0730fb00-781c-11eb-9340-3dca8c60d218.png)

Not sure that it matters in this case but worth a thought.

Reply:
ok changed

Reply:
Really, did you forget to push?

Reply:
no not pushed yet
Still looking how to deal with that "suggestion" commit

Reply:
ah ok I can apply it, pull from my box then push but need to set upstream then